content/main_tester.sh

```diff
@@ -94,7 +94,9 @@ run_submission() {
 
     case "$?" in
       "0")
-          validate_submission_output ${TEST_FDR}/outputfile_${TID}.txt ${TMP}/sub_output_${SID}_${TID}.txt
+          # validate_submission_output ${TEST_FDR}/outputfile_${TID}.txt ${TMP}/sub_output_${SID}_${TID}.txt
+          chmod +x ${PROB_FDR}/${PROB_CODE}/test_script.sh
```

Comment:
Since we are using the same program over and over again, we can move this permission modification to the top.